pkgs/crypto/keys/client/common.go

```diff
@@ -21,13 +21,18 @@ var DefaultBaseOptions = BaseOptions{
 
 func HomeDir() string {
 	// if environment set, always use that.
+	// if not, check whether can get os.UserHomeDir()
+	// if not, fall back to home directory
+	var err error
 	hd := os.Getenv("GNO_HOME")
 	if hd != "" {
 		return hd
 	}
-
-	// look for dir in home directory.
-	hd, err := os.UserHomeDir()
+	hd, err = os.UserConfigDir()
+	if err == nil {
+		return fmt.Sprintf("%s/gno", hd)
```

Comment:
```suggestion
		return filepath.Join(hd, "gno")
```

Use filepath instead of `path` or `fmt.Sprintf` when dealing with file system directories. It handles different path separators correctly.

Reply:
I will accept the change. But think should raise this as a separate issue:

Since direct slashes work in Windows (and in go), and everywhere,
can't just drop this tradition of backslashes. I see a cost, and benefit in differentiating them.

We could have a policy that says "use direct slashes, all the time". 
That way it would be simpler. Shorter code, no mental strain.

--

See this guy here for example  https://news.ycombinator.com/item?id=13950017

> (...) it's unlikely that you need to do anything at all; the Windows APIs handle forward slash as a path separator just fine.
> 
> And on that note, for any MS employees reading now and who write READMEs and other docs for projects with publicly released code in this "New Microsoft" era, please default to using forward slash for the paths in your code snippets, instead of backslash. There's hardly any reason not to. (Unless you're using the command prompt—but you guys are a pointy clicky bunch and don't like the CLI anyway, right? Even so, you have PowerShell.)
> 
> So you can drop practice of writing up separate "For Windows users" and "For Mac and Linux users" instructions that differ only in the type of path separator used. Just use plain ol' slash.

Or maybe I should open a distinct 'issue', this is not the proper place.
It's also not super important so IDK.